components/match2/commons/starcraft_starcraft2/match_summary_starcraft.lua

```diff
@@ -84,16 +84,43 @@ function StarcraftMatchSummary.Header(props)
 			showFlag = false,
 			showLink = opponent.type == 'team',
 		})
-			:addClass('brkts-popup-header-opponent')
 	end
 
-	local header = html.create('div'):addClass('brkts-popup-header-dev')
+	local headerLeft = html.create('div')
+		:addClass('brkts-popup-header-opponent')
+		:addClass('brkts-popup-header-opponent-left')
+		:css('width', '50%')
 		:node(renderOpponent(1))
+		:node(StarcraftMatchSummary._createScore(
+			match.opponents[1],
+			'left',
+			match.showScore
+		))
+	local headerRight = html.create('div')
+		:addClass('brkts-popup-header-opponent')
+		:addClass('brkts-popup-header-opponent-right')
+		:css('width', '50%')
+		:node(StarcraftMatchSummary._createScore(
+			match.opponents[2],
+			'right',
+			match.showScore
+		))
 		:node(renderOpponent(2))
+	local header = html.create('div'):addClass('brkts-popup-header-dev')
+		:node(headerLeft)
+		:node(headerRight)
 
 	return header
 end
 
+function StarcraftMatchSummary._createScore(opponent, side, showScore)
```

Comment:
I don't think this needs to be a separate function, since it's just argument passing `StarcraftOpponentDisplay.BlockScore`. Pure (stateless) display components should be public, even if used in just one module.